Source/Core/VideoBackends/Vulkan/CMakeLists.txt

```diff
@@ -1,3 +1,27 @@
+# Enable Vulkan by default everywhere except macOS, as it's not supported there.
+# It can be compiled, however there is no WSI, so it can only run headless.
```

Comment:
This comment is misleading, as not only is there no WSI on macOS, there is no Vulkan runtime. It cannot run even in headless mode.